### PR TITLE
Support for SSH agent forwarding

### PR DESCRIPTION
**Pull request by montdidier**
I am currently adding support for SSH agent forwarding in phpseclib with the goal of providing SSH agent forwarding support in rocketeer. As this library is a dependency of rocketeer I need to add support for agent forwarding here as well. This is my first run at it. Comments and feedback please. 

See https://github.com/phpseclib/phpseclib/pull/518 for agent forwarding support in phpseclib.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "target-dir": "Illuminate/Remote",
     "extra": {
         "branch-alias": {
-            "dev-master": "4.2-dev"
+          "dev-master": "4.2-dev"
```
Comment (GrahamCampbell) [inline composer.json]: Please revert this.

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Comment by montdidier** [comment]
ok? although any release will need to come after phpseclib part I figure.

**Comment by dinamic** [comment]
Hey @montdidier, the support in phpseclib seem to have been implemented back at late March.

PR: https://github.com/phpseclib/phpseclib/pull/592

Could you please add the support in Rocketeer? Thanks.

**Comment by GrahamCampbell** [comment]
Please close this.

**Comment by GrahamCampbell** [comment]
This repo is read only, and this will never be merged.

**Pull request closed without merging**